sos/report/plugins/ext.py

```diff
@@ -0,0 +1,51 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Ext(Plugin, IndependentPlugin):
+    """This plugin collects information on mounted Ext2/3/4 filessystems on
+    the local system.
+
+    Users should expect `dumpe2fs -h` or `dumpe2fs` collections by this
+    plugin for each Ext2/3/4 filesystem that is recognized by lsblk.
+    """
+
+    short_desc = 'Ext2/3/4 filesystem'
+
+    plugin_name = 'ext4'
+    profiles = ('storage')
+    files = ('/sys/fs/ext4/', '/proc/fs/ext4/', '/proc/fs/jbd2/')
+
+    def setup(self):
+        dumpe2fs_opts = '-h'
+        if self.get_option('dumpe2fs'):
```

Comment:
The options are only available it they have been defined, similar to the `filesys.py`